frontend/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -181,7 +176,34 @@ const childColumns = [
     title: "Description",
     dataIndex: "description",
     key: "description",
-    render: (text) => <div title="Description">{text}</div>,
+    render: (text) => (
+      <div title="Description" style={{ maxWidth: "800px" }}>
+        {text}
+      </div>
+    ),
+  },
+  {
+    title: "Files",
+    dataIndex: "documents",
+    key: "documents",
+    render: (text) => (
+      <div>
```

Comment:
This may have been intentional, and if so, disregard. 
 But the title attribute on the outer div is missing and should be added for the responsive card.

Reply:
No, I think this was missed. Added! 👍